### DISCUSSION TITLE
Error when adding Quasar to Vue3 application

### DISCUSSION THREAD

**Discussion by peppermintbunny** [quasarframework/quasar#15763] [General - Components / Directives / etc] [answered]
I have a new Vue 3 app with VITE v4.3.3 and wanted to add Quasar (which ( love using) but have come up difficulties.  Please can someone help me.
I checked and I have @vue/cli 5.0.8
I ran: 'vue add quasar' and was told 'Installing vue-cli-plugin-quasar...'  I selected No to replace app.vue etc; selected Sass; selected Material Icons; selected en-US;No RTL;Select features: Roboto font, Material, Material Round, MDI
I then ran 'npm run dev' and got error - [sass] Can't find stylesheet to import.

Please suggest how I can correct this.

Many thanks

**Comment by smolinari** [comment]
@peppermintbunny - Is the Vue 3 app already a developed app, or are you just starting off with it?

Scott

**Reply by peppermintbunny** [reply]
Hi Scott
I was just creating a new Vue3 app. I was hoping to create a Vue 3 app and link quasar 2 to it. To. be honest, I didnt understand the page in the new documentation entitled: UMD/Standalone so followed the instruction on the page Vite Plugin.  
What I'm trying to do is find a way to create and use global variables for a few bits of information I need throughout my app.  In the past I've used the Quasar CLI but I cant find a way to use globalProperties. All the example I can find assume one has access to the app object on a main.js page. Hence the pickle I've landed myself in.

**Reply by yusufkandemir** [reply, marked as answer]
@peppermintbunny boot files is a valid use for adding stuff to `globalProperties` which is mentioned in the docs: https://quasar.dev/quasar-cli-vite/boot-files#examples-of-appropriate-usage-of-boot-files
There is also an example down on the same page: https://quasar.dev/quasar-cli-vite/boot-files#axios
So, you can keep on using Quasar CLI.

**Reply by peppermintbunny** [reply]
Thank you!! I've just looked at the example you have pointed to and I feel embarrassed at having missed this!  I have found that this new version of the documentation is so very helpful.  I keep finding explanation for things that I have struggled with in the past.

**Reply by smolinari** [reply]
The rule of thumb with Quasar is, the best experience you can get with Quasar will always be with Quasar's own CLI (no matter what!). Tons of thought have been put into the CLI to make you, the developer, the most productive you can be. Going any other route with Quasar will just mean more effort to get anything done.  

Also a rule of thumb, always explain the actual problem you haven't solved yet when presenting your problem. That will let us help you get to the solution faster too. :grin: 

Scott
